ciborium/tests/codec.rs

```diff
@@ -416,3 +416,18 @@ fn byte_vec_serde_bytes_compatibility(input: Vec<u8>) {
     let bytes: Vec<u8> = from_reader(&buf[..]).unwrap();
     assert_eq!(input, bytes);
 }
+
+#[derive(Serialize, Deserialize, Eq, PartialEq, Debug)]
+struct Foo {
+    bar: u8,
+}
+
+#[rstest(input, expected,
+    case("a163626172182a", Foo { bar: 42 }),
+    case("a143626172182a", Foo { bar: 42 }),
+)]
+fn handle_struct_field_names(input: &str, expected: Foo) {
+    let mut buf = hex::decode(input).unwrap();
```

Comment:
Seems this doesn't need to be mutable, could you remove `mut` to get rid  of the warnings?

Reply:
Sure thing